### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by LucasCairns**
## Context

Whilst getting set up, noticed we hadn't got a complete `README.md`. I've took a bit of inspiration from our other Backend/Kotlin repositories - updating where required

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,97 @@
...
+The generated documentation for the api can be viewed at http://localhost:8080/swagger-ui.html
+
+### Health
+
+- `/ping`: will respond `pong` to all requests.  This should be used by dependent systems to check connectivity to   
+  offender assessment service, rather than calling the `/health` endpoint.
+- `/health`: provides information about the application health and its dependencies.  This should only be used  
+  by offender assessment service health monitoring (e.g. pager duty) and not other systems who wish to find out the   
+  state of offender assessment service.
+- `/info`: provides information about the version of deployed application.
```
Comment (wood-n) [inline README.md:67]: I'm not sure if there is an /info endpoint

File: README.md [hunk 2]
```diff
@@ -1,11 +1,97 @@
...
+
+#### Default port
+By default the application starts on port '8080'.   To override, set server.port (e.g. `SERVER_PORT=8099 java -jar build/libs/offender-assessments-api-<yyyy-mm-dd>.jar` )
+
+### Documentation
+The generated documentation for the api can be viewed at http://localhost:8080/swagger-ui.html
+
+### Health
+
+- `/ping`: will respond `pong` to all requests.  This should be used by dependent systems to check connectivity to
```
Comment (wood-n) [inline README.md:62]: I think the ping endpoint is at health/ping

File: README.md [hunk 3]
```diff
@@ -1,11 +1,97 @@
...
+java -jar build/libs/offender-assessments-events-<yyyy-mm-dd>.jar  
+```
+
+### Start the application with H2 database and LocalStack configuration
+
+The application can be configured to use LocalStack and an in-memory H2 DB by using the spring boot profile `local`. Ensure you have LocalStack running, on the command line run:
+```  
+SPRING_PROFILES_ACTIVE=dev AWS_PROVIDER=localstack AWS_ACCESS_KEY=local AWS_SECRET_KEY=dev
+java -jar build/libs/offender-assessments-events-<yyyy-mm-dd>.jar  
+```
```
Comment (wood-n) [inline README.md:48]: Might need to add to run the localstack script?

**Review by wood-n — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (LucasCairns) [reply README.md:52]: Good shout, I've added this to the `backing services` section of the document

File: README.md (hunk 2 above)
Comment (LucasCairns) [reply README.md]: Updated, this should now have the correct path `/health/ping`

File: README.md (hunk 1 above)
Comment (LucasCairns) [reply README.md:71]: Just had a quick check ,`/info` is returning

**Review by adelaidaa — APPROVED** [review]

**Pull request merged**